client/src/components/common/Dialog.js

```diff
@@ -0,0 +1,49 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material'
+import PropTypes from 'prop-types'
+
+ConfirmationDialog.propTypes = {
+  open: PropTypes.bool.isRequired,
+  handleClose: PropTypes.func.isRequired,
+  handleConfirm: PropTypes.func.isRequired,
+  title: PropTypes.string,
+  description: PropTypes.string,
+}
+
+export function ConfirmationDialog({
+  open,
+  handleClose,
+  handleConfirm,
+  title,
+  description,
+}) {
+  return (
+    <Dialog
```

Comment:
Is there a way to set MUI elements to be dark mode?

Reply:
And do we want to just use the modal that u made instead?

Reply:
youre right, i forgot i made a modal

Reply:
for the dark mode, i think we need to configure the app.js so that it applies to the mui components in it. https://mui.com/material-ui/customization/dark-mode/